File: ToyLanguage/src/Model/ConstExp.java
package Model;

public class ConstExp extends Exp {
    private int number;

    public ConstExp(int number) {
        this.number = number;
    }

    int eval(MyIDictionary<String,Integer> tbl) {return number;}

    @Override
    public String toString() {
        return "ConstExp{" +
                "number=" + number +
                '}';
    }
}

File: ToyLanguage7/ToyLanguage7/ConstExp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ToyLanguage7
{
    class ConstExp : Exp
    {
        private int number;

        public ConstExp(int number)
        {
            this.number = number;
        }

        public override int eval(MyIDictionary<String, int> tbl) { return number; }
        
        public override string ToString()
        {
            return number.ToString();
        }

    }
}
